manifests/base/manager/manager.yaml

```diff
@@ -23,6 +23,14 @@ spec:
       containers:
         - name: manager
           image: ghcr.io/kubeflow/trainer/trainer-controller-manager
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsNonRoot: true
```

Comment:
What about: readOnlyRootFilesystem ?

Reply:
That depends on you. I think it is not required for PSS restricted.